osm_validator/views.py

```diff
@@ -1,11 +1,21 @@
+import time
+
 from aiohttp import web
+from aiohttp_session import get_session
 from psycopg2._psycopg import IntegrityError
 
 from . import models
 from .oauth import OSMOauthClient
 
 
 async def index(request):
+    sesion = await get_session(request=request)
+    user_uid = sesion.get('message')
+    user = sesion['user'] if 'user' in sesion else None
```

Comment:
it's pretty boilerplate code, lets move it to middleware https://docs.aiohttp.org/en/stable/web.html#middlewares where we will set `request.user`